app/helpers/users_with_access_helper.rb

```diff
@@ -0,0 +1,29 @@
+module UsersWithAccessHelper
+  def formatted_user_name(user)
+    status = if user.invited_but_not_yet_accepted?
+               " (invited)"
+             elsif user.suspended?
+               " (suspended)"
+             elsif user.access_locked?
+               " (access locked)"
+             else
+               ""
+             end
+
+    "#{link_to(user.name, edit_user_path(user))}#{status}".html_safe
+  end
```

Comment:
Minor: It might be nice to add a unit test for this helper method and possibly the other helper methods you've added since they aren't completely trivial logic-wise.

Reply:
Thanks James, that's a good suggestion. I'll rewrite the history to add the tests at the appropriate points.